app/server.js

```diff
@@ -1,12 +1,12 @@
 "use strict";
 
-const http = require("http"),
+const https = require("https"),
   express = require("express");
 
 const ServerFactory = function(config, logger) {
 
   const app = express(),
-    server =  http.createServer(app);
```

Comment:
@stulees I had a read and I am not sure we should allow http. We are changing to https to avoid issues so if we allow these issues may come back from the community for instance. Besides, adding a proxy will add complexity to RC in which increases the maintenance effort. Also, we are going against the service specifications which says a unique port for HTTP and HTTPS.

@tejohnso @alanclayton what do you think? cheers